calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -220,10 +228,14 @@ def on_evaluation_sample_changed(index):
             img = self.data_collector.get_evaluation_image(index)
             self.process_db_data(img)
 
+        def on_rectify_type_change(index):
+            self.calibrated_camera_model._cached_undistorted_model = None
```

Comment:
Variables with `_` as a prefix indicate that they should not be modified from the outside